npp.sls

```diff
@@ -6,6 +6,13 @@
 {% endif %}
 npp:
   {% for version in [
+    '8.6.2',
+    '8.6.1',
+    '8.6',
+    '8.5.8',
+    '8.5.7',
+    '8.5.6',
+    '8.5.5',
```

Comment:
The URL tests fail on `8.5.5`. It doesn't have a download artifact. You can either not add this version, or you can move it to the bottom list.